tests/kernels/utils.py

```diff
@@ -10,10 +10,12 @@
 import torch
 
 from vllm.attention import AttentionBackend, AttentionMetadata, AttentionType
-from vllm.attention.backends.xformers import XFormersBackend
-from vllm.utils import (STR_BACKEND_ENV_VAR, STR_XFORMERS_ATTN_VAL,
+from vllm.utils import (STR_BACKEND_ENV_VAR, STR_XFORMERS_ATTN_VAL, is_hip,
                         make_tensor_with_pad)
 
+if not is_hip():
+    from vllm.attention.backends.xformers import XFormersBackend
+
```

Comment:
Could you please do lazy import in L522?
```python
    if backend_name == STR_XFORMERS_ATTN_VAL:
        # NOTE: xFormers backend cannot be imported for AMD GPUs.
        from vllm.attention.backends.xformers import XFormersBackend
        return XFormersBackend()
```

Reply:
done